1.9/usage/gpu-support.md

```diff
@@ -0,0 +1,224 @@
+AWS CloudFormation templates now available at: https://<amazon-web-endpoint>/<path-to-directory>
+
+#### Create Dependencies
+
+Use the `zen.sh` script to create the Zen template dependencies. These dependencies will be used as input to create your stack in CloudFormation.
+
+1. Save the following script as zen.sh.
+
+    ```bash
+    #!/bin/bash
+    export AWS_DEFAULT_OUTPUT="json"
+    set -o errexit -o nounset -o pipefail
+
+    if [ -z "${1:-}" ]
+    then
+      echo Usage: $(basename "$0") STACK_NAME
+      exit 1
+    fi
+
+    STACK_NAME="$1"
+    VPC_CIDR=10.0.0.0/16
+    PRIVATE_SUBNET_CIDR=10.0.0.0/17
+    PUBLIC_SUBNET_CIDR=10.0.128.0/20
+
+    echo "Creating Zen Template Dependencies"
+
+    vpc=$(aws ec2 create-vpc --cidr-block "$VPC_CIDR" --instance-tenancy default | jq -r .Vpc.VpcId)
+    aws ec2 wait vpc-available --vpc-ids "$vpc"
+    aws ec2 create-tags --resources "$vpc" --tags Key=Name,Value="$STACK_NAME"
+    echo "VpcId: $vpc"
+
+    ig=$(aws ec2 create-internet-gateway | jq -r .InternetGateway.InternetGatewayId)
+    aws ec2 attach-internet-gateway --internet-gateway-id "$ig" --vpc-id "$vpc"
+    aws ec2 create-tags --resources "$ig" --tags Key=Name,Value="$STACK_NAME"
+    echo "InternetGatewayId: $ig"
+
+    private_subnet=$(aws ec2 create-subnet --vpc-id "$vpc" --cidr-block "$PRIVATE_SUBNET_CIDR" | jq -r .Subnet.SubnetId)
+    aws ec2 wait subnet-available --subnet-ids "$private_subnet"
+    aws ec2 create-tags --resources "$private_subnet" --tags Key=Name,Value="${STACK_NAME}-private"
+    echo "Private SubnetId: $private_subnet"
+
+    public_subnet=$(aws ec2 create-subnet --vpc-id "$vpc" --cidr-block "$PUBLIC_SUBNET_CIDR" | jq -r .Subnet.SubnetId)
+    aws ec2 wait subnet-available --subnet-ids "$public_subnet"
+    aws ec2 create-tags --resources "$public_subnet" --tags Key=Name,Value="${STACK_NAME}-public"
+    echo "Public SubnetId: $public_subnet"
+    ```
+
+1. Run the script from from your local terminal with a single argument of stack prefix (STACK_NAME).
+    ```bash
+    $ bash ./zen.sh dcos
+    ```
+    
+    The output should look like this:
+    
+    ```
+    Creating Zen Template Dependencies
+    VpcId: vpc-e0bd2c84
+    InternetGatewayID: igw-38071a5d
+    Private SubnetId: subnet-b32c82c5
+    Public SubnetId: subent-b02c55c4
+    ```
+    
+1. Use these dependency values as input to create your stack in CloudFormation in the next steps.
+
+#### Deploy the CF template
+
+1. Go to [CloudFormation](https://console.aws.amazon.com/cloudformation/home) and click **Create Stack**.
+
+1. On the **Select Template** page, specify the custom template that you created on your master node via the `sudo bash dcos_generate_config.sh --aws-cloudformation` command . For example, `
+https://s3-us-west-2.amazonaws.com/joel-test-gpu/templates/dcos/config_id/fe4f3d47e23eda85301c875a28a34a8f48e44f95/cloudformation/el7-zen-1.json`.
+
+1. On the **Specify Details** page, specify these values and and click **Next**.
```

Comment:
@sascala all of the parameters are still shown https://cl.ly/1Z2M0J2t3v0o? Please see my suggestion above.